site/app/models/Email.php

```diff
@@ -22,8 +22,8 @@ class Email extends AbstractModel {
     protected $subject;
     /** @property @var string Body of email */
     protected $body;
-    /** @property @var string user name */
-    protected $user_id;
+    /** @property @var string sender's user id */
+    protected $sender_id;
```

Comment:
see discussion in previous comment. `user_id` marks who the email is going to, and recipient is a legacy field that currently holds the value of `user_email` for whoever `user_id` is.